src/components/UIShell/PlatformHeaderNavMenu.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { ChevronDownGlyph } from '@carbon/icons-react';
+
+const PlatformHeaderNavMenuItem = ({ title }) => {
+  return (
+    <li className="bx--platform-header__dropdown-item">
+      {title}
+      <ChevronDownGlyph className="bx--platform-header__arrow" />
+      <ul className="bx--dropdown-item__menu" role="menubar" aria-label={title}>
+        <li class="bx--dropdown-item__menu-item">
```

Comment:
Is this component for demonstration purpose? (Or will the content here be replaced with something else...?)

Reply:
@asudoh honestly is just WIP code as I was copying over from `carbon-components`. I noticed that the snippet wasn't accessible though so I stopped working on it and made https://github.com/IBM/carbon-components/issues/1270 to tackle before finishing up work here 👍